src/main/java/com/weirddev/testme/intellij/template/context/PowerMockBuilder.java

```diff
@@ -0,0 +1,43 @@
+package com.weirddev.testme.intellij.template.context;
+
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+public class PowerMockBuilder extends MockitoMockBuilder{
+    /**
+     * true when render internal method call option is opted-in on custom settings
+     */
+    private final boolean renderInternalMethodCallStubs;
+
+    public PowerMockBuilder(boolean isMockitoMockMakerInlineOn, boolean stubMockMethodCallsReturnValues,
+        TestSubjectInspector testSubjectInspector, @Nullable String mockitoCoreVersion,
+        boolean renderInternalMethodCallStubs) {
+        super(isMockitoMockMakerInlineOn, stubMockMethodCallsReturnValues, testSubjectInspector, mockitoCoreVersion);
+        this.renderInternalMethodCallStubs = renderInternalMethodCallStubs;
+
+    }
+
+    /**
+     *
+     * @param method call method
+     * @param testedClass the tested class
+     * @return true - if the method is  class object self called method
+     */
+    public boolean isMethodOwnedByClass(Method method, Type testedClass) {
```

Comment:
this util method would be more relevant if moved to _TestSubjectInspector_ (exposed in templates as TestSubjectUtils). please also move existing methods isNotInjectedInDiClass & hasAccessibleCtor to _TestSubjectInspector_